db/dbSeed.js

```diff
@@ -1,7 +1,145 @@
+  connection.query('TRUNCATE TABLE products', (err, results, fields) => {
+    if (err) {
+      console.log(err);
+      return;
+    }
+    console.log('Records deleted in table: products, AUTO_INCREMENT reset to 0');
+  });
+  var i = 0;
+  while (i < 100) {
+    genNewProduct();
+    i++;
+  }
+  console.log('Products table has been reset and seeded.');
+};
+
+////////////////////////////////////////
+//    setUpImagesTable: clears the product table and restarts auto_increment @ 0.
+////////////////////////////////////////
+//product photos url: https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/[1-500].jpeg
+//var prodPhotosBase = `https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/${num}.jpeg`; //[1-500].jpeg
+var setUpImagesTable = function () {
+  var urls = [];
+  var baseUrl = 'https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/';
+  for (var i = 1; i <= 500; i++) {
+    urls.push(baseUrl + `${i}.jpeg`);
+  }
+
+  connection.query('TRUNCATE TABLE images', (err, results, fields) => {
+    if (err) {
+      console.log(err);
+      return;
+    }
+    console.log('Records deleted in table: images, AUTO_INCREMENT reset to 0');
+  });
+  var i = 0;
+  for (var i = 1; i <= 100; i++) {
+    for (var j = 1; j <= 5; j++) {
+      urlField = urls.pop();
+      params = [i, urlField];
+      connection.query('INSERT INTO images (product_id, url) VALUES (?,?)', params, (err, results, fields) => {
+        if (err) {
+          console.log(err);
+        }
+      });
+    }
+  }
+  console.log('Images table reset and seeded.');
+};
+
+////////////////////////////////////////
+//    setUpESRBRatingsTable: clears the esrbRatings table and restarts auto_increment @ 0.
+////////////////////////////////////////
+//product photos url: https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/[1-500].jpeg
+//var prodPhotosBase = `https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/product/${num}.jpeg`; //[1-500].jpeg
+var setUpESRBRatingsTable = function () {
+  var urls = [];
+  var baseUrl = 'https://hrr45-fec.s3.us-east-2.amazonaws.com/photos/esrb/';
+  for (var i = 1; i <= 7; i++) {
+    urls.push(baseUrl + `${i}.jpeg`);
+  }
+
+  var ratingNames = ['early-childhood', 'everyone', 'everyone10+', 'teen', 'mature', 'adults-only', 'rating-pending'];
+
+  connection.query('TRUNCATE TABLE esrbRatings', (err, results, fields) => {
+    if (err) {
+      console.log(err);
+      return;
+    }
+    console.log('Records deleted in table: esrbRatings, AUTO_INCREMENT reset to 0');
+  });
+
+  for (var i = 0; i < 7; i++) {
+    params = [ratingNames[i], urls[i]];
+    connection.query('INSERT INTO esrbRatings (name, url) VALUES (?,?)', params, (err, results, fields) => {
+      if (err) {
+        console.log(err);
+      }
+    });
+  }
+  console.log('esrbRatings table reset and seeded.');
+};
+
+
+//call reset functions
+setUpProductTable();
+setUpImagesTable();
+setUpESRBRatingsTable();
+
+//end connection
+connection.end();
```

Comment:
Both git and airbnb are gonna harass you about needing a newline at the end of every script.

Reply:
I need to get Airbnb style guide going.